09-workload.md

```diff
@@ -6,37 +6,48 @@ The cluster now has an [Traefik configured with a TLS certificate](./09-secret-m
 
 > :book: The Contoso app team is about to conclude this journey, but they need an app to test their new infrastructure. For this task they've picked out the venerable [ASP.NET Core Docker sample web app](https://github.com/dotnet/dotnet-docker/tree/master/samples/aspnetapp).
 
-1. Deploy the ASP.NET Core Docker sample web app
+1. Deploy the ASP.NET Core Docker sample web app in the AKS Cluster deployed to Region 1
 
    > The workload definition demonstrates the inclusion of a Pod Disruption Budget rule, ingress configuration, and pod (anti-)affinity rules for your reference.
 
    ```bash
-   kubectl create -f ./workload/aspnetapp.yaml --context $AKS_CLUSTER_NAME_BU0001A0042_03
+   kubectl apply -f ./workload/aspnetapp.yaml --context $AKS_CLUSTER_NAME_BU0001A0042_03
    ```
 
-1. Wait until is ready to process requests running
+1. Deploy the same workload as another instance in the second AKS cluster deployed to Region 2. Additionally, you will be enable some canary testing from the second cluster.
+
+   > :book: The app team is now more confident than ever with its second regional AKS cluster. The team follows an active/active High Availability strategy, and it is known that the major stream of clients comes from East US 2. Therefore, they realize they have some iddle resources most of the time from Central US. This makes them to consider starting some canary testing in that specific region as this will be introducing a very low risk for the normal operatory. Using a weighted load balanced strategy, clients will be attended most of the time by their well known and stable ASP.NET Core 3.1 workload app, while others by new ASP.NET 5 aleatory. From this experimentation, the app team wants to evaluate the memory utlization of their workload using the new ASP.NET version 5. This will allow them to plan ahead before a full migration.
```

Comment:
```suggestion
   > :book: The app team is now more confident than ever with its second regional AKS cluster. The team follows an active/active High Availability strategy, and it is known that the major stream of clients comes from East US 2. Therefore, they realize they have some idle resources most of the time from Central US. This makes them to consider starting some canary testing in that specific region as this will be introducing a very low risk for normal operations. Using a weighted load balanced strategy, clients will be attended most of the time by their well known and stable ASP.NET Core 3.1 workload app, while others by new ASP.NET 5 aleatory. From this experimentation, the app team wants to evaluate the memory utilization of their workload using the new ASP.NET version 5. This will allow them to plan ahead before a full migration.
```